src/System.Text.Encodings.Web/tests/JavaScriptStringEncoderTests.cs

```diff
@@ -81,21 +81,28 @@ public void Default_ReturnsSingletonInstance()
             Assert.Same(encoder1, encoder2);
         }
 
-        // TODO: fix xUnit escaping and then uncomment
-        //[Theory]
-        //[InlineData("<", @"\u003C")]
-        //[InlineData(">", @"\u003E")]
-        //[InlineData("&", @"\u0026")]
-        //[InlineData("'", @"\u0027")]
-        //[InlineData("\"", @"\u0022")]
-        //[InlineData("+", @"\u002B")]
-        //[InlineData("\\", @"\\")]
-        //[InlineData("/", @"\/")]
-        //[InlineData("\b", @"\b")]
-        //[InlineData("\f", @"\f")]
-        //[InlineData("\n", @"\n")]
-        //[InlineData("\t", @"\t")]
-        //[InlineData("\r", @"\r")]
+        [Fact]
+        public void JavaScriptStringEncode_AllRangesAllowed_StillEncodesForbiddenChars_Simple_Escaping() {
+            // The following two calls could be simply InlineData to the Theory below
+            // Unfortunatelly, the xUnit logger fails to escape the inputs when loffing the test results,
```

Comment:
supposedly "loff" is an internet slang for "love" :-)

... good catch! will fix.